app/clients/sms/twilio.py

```diff
@@ -216,7 +216,7 @@ def translate_delivery_status(
         - record_status: the convereted twilio to notification platform status
         - reference: the message id of the twilio message
         - payload: the original payload from twilio
-        https://github.com/department-of-veterans-affairs/vanotify-team/blob/master/Engineering/SPIKES/SMS-Delivery-Status.md#twilio-implementation-of-delivery-statuses
```

Comment:
I am aware this is links references the `vanotify-team` repo. We will be updating `master` to `main` in ticket [#1351](https://app.zenhub.com/workspaces/va-notify-620d21369d810a00146ed9c8/issues/gh/department-of-veterans-affairs/vanotify-team/1351). Since this is only documentation, figured we're OK to update it now.

Reply:
Add AC to [#1351](https://app.zenhub.com/workspaces/va-notify-620d21369d810a00146ed9c8/issues/gh/department-of-veterans-affairs/vanotify-team/1351) to confirm links work